tests/urlpatterns/tests.py

```diff
@@ -193,6 +206,42 @@ def test_invalid_converter(self):
         with self.assertRaisesMessage(ImproperlyConfigured, msg):
             path("foo/<nonexistent:var>/", empty_view)
 
+    def test_invalid_override_default_converter(self):
+        msg = "Converter int is already registered. "
+        "Support for overriding registered converters is deprecated "
+        "and will be removed in Django 6.0."
```

Comment:
Without parentheses, the other strings are not being concatenated. They’re just standalone lines that contain only a string constant.
```suggestion
        msg = (
            "Converter int is already registered. "
            "Support for overriding registered converters is deprecated "
            "and will be removed in Django 6.0."
        )
```